packages/portal-proto/src/features/GenomicTables/GenesTable/utils.tsx

```diff
@@ -56,11 +56,12 @@ export const useGenerateGenesTableColumns = ({
             classNames={{
               input: "checked:bg-accent checked:border-accent",
             }}
+            aria-label={`checkbox for selecting genes table rows currently displayed`}
```

Comment:
this is unhelpful to someone who cannot see the page

Reply:
true, changed to include comma separated list of the unique identifiers

Reply:
ideally you would want something like "select all rows, (number of rows) rows on page x of x"